guide.md

```diff
@@ -0,0 +1,406 @@
+Getting Started 2.0
+===
+
+For this guide, we're going to walk through some simple examples for each of the
+pieces of CRUD. Each step in this guide will build on the previous, and is meant
+to be followed along. This guide assumes that you're using PostgreSQL on
```

Comment:
`is meant to be follow along.` feels a little awkward. Maybe reword to `is meant to be followed in order.`?